submissions/YuliiaHordiichuk/a-tiny-js-world/index.js

```diff
@@ -0,0 +1,75 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: https://github.com/YuliiaHordiichuk/a-tiny-JS-world
+   Web app: https://yuliiahordiichuk.github.io/a-tiny-JS-world/
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+
+const man = {
+   species: 'human',
+   name: 'Mark',
+   gender: 'male',
+   legs: 2,
+   hands: 2,
+   saying: 'Hi, there!'
+};
+
+const woman = {
+   species: 'human',
+   name: 'Jane',
+   gender: 'female',
+   legs: 2,
+   hands: 2,
+   saying: 'Hello, darling!'
+};
+
+const dog = {
+   species: 'dog',
+   name: 'Donny',
+   gender: 'male',
+   legs: 4,
+   hands: 0,
+   saying: 'Woof!'
+};
+
+const cat = {
+   species: 'cat',
+   name: 'Beniia',
+   gender: 'male',
+   legs: 4,
+   hands: 0,
+   saying: 'Mrrrr!'
+};
+
+// ======== OUTPUT ========
+
+const inhabitants = [man, woman, dog, cat]; 
+
+let message = (obj) => {
+   return `${obj.saying} I am a ${obj.species} and my name is ${obj.name}. Nice to meet you! My gender is ${obj.gender} so I have ${obj.legs} legs and ${obj.hands} hands.`;  
+} 
```

Comment:
It's a pretty long string here. If we want to add another 5 or 6 properties it will become longer and not really easy to read, comprehend and expand even more.
How about splitting it into logical parts, putting them into a nicely formatted array and then gluing them all with an `Array.join`?
This way we could easily add substrings for new properties and reorder them with lesser effort.

Reply:
Thanks for your comment, I have done this :)